src/services/positionService.ts

```diff
@@ -713,6 +734,77 @@ export default class PositionService {
     );
   }
 
+  async buildXcallDepositParams(
+    fromToken: Token,
+    toToken: Token,
+    fromValue: string,
+    frequencyType: BigNumber,
+    frequencyValue: string,
+    yieldFromPossible?: string,
+    yieldToPossible?: string,
+    destinantionChainID?: number
+  ) {
+    const destinantionUSDC = this.connextService.getNativeUSDCAddress(destinantionChainID as number);
+
+    const { forwardCallData } = await this.buildDepositParams(
+      fromToken,
+      toToken,
+      fromValue,
+      frequencyType,
+      frequencyValue,
+      yieldFromPossible,
+      yieldToPossible,
+      destinantionUSDC
+    );
+
+    const xTargetAddress = '0xC825d25123Ca8d49066cf8F0AeE164660056e172'; // polygon address for mean target
```

Comment:
Could we define a constant in addresses that is a map of chainId to this addresses?